### PR TITLE
docs(customise sass page): add guides, update config

### PR DESCRIPTION
**Pull request by rav3**

### CODE REVIEW

**Review by clukhei — APPROVED** [review]
rmb to hide patterns

File: _config.yml [hunk 1]
```diff
@@ -7,16 +7,16 @@ logo: /assets/img/logo_sgds.png
...
+sgds-version: 2.0.0
 report-vulnerability: "https://tech.gov.sg/report_vulnerability"
 contact-us: "https://form.gov.sg/forms/govtech/5bc440ac507956000fdbae91"
 feedback-form: "https://form.gov.sg/#!/forms/govtech/5bd12c6668dbe7000fff9ace"
 request-component: "https://form.gov.sg/6136ea11658bea0012fc9e80"
 search_placeholder: "What are you looking for?"
-react-storybook: https://1-0-0-dev.dg2jutx9gylat.amplifyapp.com/
-github-link: https://github.com/GovTechSG/sgds/tree/2.0.0-dev
+react-storybook: https://react.designsystem.tech.gov.sg
+github-link: https://github.com/GovTechSG/sgds/tree/v2
```
Comment (haphizi) [inline _config.yml:17]: change to https://github.com/GovTechSG/sgds/tree/v2dev

File: collections/_get-started/customise-with-sass.html [hunk 2]
```diff
@@ -1,33 +1,128 @@
...
+    </section>
+    <section name="section-design-tokens">
+      <h2 class="title fw-bold" id="anatomy">Variables</h2>
+      <p>
+        You can find the complete list of <code>sgds-theme</code> variables inside <code>sass/sgds-theme/_sgds-variables.scss</code> or go to our github page <a href="{{site.sgds-variables-github}}" target="_blank" rel="noopener noreferrer">here</a>
+      </p>
+      
+      
+      <div class="section-sub">
+        <p>Here's an example that changes the background-color and color for the <body> when importing and compiling <code>sgds-theme</code> via npm:</p>
```
Comment (haphizi) [inline collections/_get-started/customise-with-sass.html:124]: color for the ...<body>.. not seen in browser

File: _config.yml (hunk 1 above)
Comment (rav3) [reply _config.yml:17]: I've placed in the `/v2/` so we wont have to update during launch

Comment (haphizi) [reply _config.yml:17]: oh okies noted

File: collections/_get-started/customise-with-sass.html (hunk 2 above)
Comment (rav3) [reply collections/_get-started/customise-with-sass.html]: content updated in commit [0388edf](https://github.com/GovTechSG/sgds/pull/369/commits/0388edfb0e24d0c636661b6bdbf253f8f94ff4e1)

**Review by haphizi — APPROVED** [review]

**Pull request merged**